tests/consent_during_vaccination.spec.ts

```diff
@@ -22,6 +24,11 @@ async function given_the_app_is_setup() {
   await p.goto("/reset");
 }
 
+async function and_i_am_signed_in() {
+  await p.goto("/users/sign_in");
+  signInTestUser(p);
```

Comment:
I think these should all be awaits:

```suggestion
  await signInTestUser(p);
```